### PR TITLE
feat: add OP Sepolia 1.3.0 and 1.4.1 safe contracts

### PR DESCRIPTION
**Pull request by ElvisKrop**
- https://chainlist.org/chain/11155420
- https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-11155420.json
- [OP Sepolia Explorer](https://optimism-sepolia.blockscout.com/)

### CODE REVIEW

File: gnosis/safe/addresses.py [hunk 1]
```diff
@@ -686,6 +686,30 @@
             "1.3.0",
         ),  # default singleton address
     ],
+    EthereumNetwork.OP_SEPOLIA_TESTNET: [
+        ("0x29fcB43b46531BcA003ddC8FCB67FFE91900C762", 7162879, "1.4.1+L2"),
```
Comment (falvaradorodriguez) [inline gnosis/safe/addresses.py:690]: Please add first the addresses on https://github.com/safe-global/safe-deployments/tree/main for v1.4.1

Comment (falvaradorodriguez) [inline gnosis/safe/addresses.py:690]: Please add first the addresses on https://github.com/safe-global/safe-deployments/tree/main for v1.4.1

**Review by falvaradorodriguez — CHANGES_REQUESTED** [review]

File: gnosis/safe/addresses.py (hunk 1 above)
Comment (ElvisKrop) [reply gnosis/safe/addresses.py:690]: :wink: 
- https://github.com/safe-global/safe-deployments/issues/392
- https://github.com/safe-global/safe-deployments/issues/393

**Review by falvaradorodriguez — DISMISSED** [review]

**Comment by ElvisKrop** [comment]
@falvaradorodriguez done :heavy_check_mark:

**Review by falvaradorodriguez — APPROVED** [review]

**Pull request merged**